NuRadioMC/simulation/simulation.py

```diff
@@ -1115,11 +1121,6 @@ def _get_em_had_fraction(self, inelasticity, inttype, flavor):
         elif(inttype == 'em'):
             fem = 1
             fhad = 0
-        elif(np.abs(flavor) == 15):
```

Comment:
doesn't these lines mean that for taus it was already simulated correctly despite the bug in particle codes?

Reply:
No, because the tau_e and tau_had inttypes were related to the old continuous slowing down approximation for taus, so the result was that fem and fhad were zero. The flow of this function wasn't prepared for these kind of faulty input files.